WMIAdventure/frontend/src/js/components/profile/molecules/UpgradeCardSection/UpgradeCardSection.js

```diff
@@ -156,7 +156,7 @@ class UpgradeCardSection extends React.Component {
     }
 
     canUpgrade() {
-        if (!this.state.userPoints) return true;
+        if (this.state.userPoints === null || this.state.userPoints === undefined) return true;

```

Comment:
czemu tutaj zwracane jest `true`, gdy `userPoints` nie ma wartości?

Reply:
Bo jak dawałem na `false` to buttony mi na chwile mrugały zawsze przy loadzie componentu